api/src/main/java/org/motechproject/nms/api/web/StatusController.java

```diff
@@ -0,0 +1,29 @@
+package org.motechproject.nms.api.web;
+
+import org.motechproject.alerts.contract.AlertCriteria;
+import org.motechproject.alerts.contract.AlertService;
+import org.motechproject.alerts.domain.Alert;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.List;
+
+@Controller
+public class StatusController {
+    @Autowired
+    private AlertService alertService;
+
+    @RequestMapping("/status")
+    @ResponseBody
+    public String status() {
+        List<Alert> alerts = alertService.search(new AlertCriteria());
```

Comment:
Will the blank AlertCriteria return all alerts?  Don't we want just those that are not Closed?